<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_12">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>product list</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTreeView" name="treeView">
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>product info</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="Agenda">
           <attribute name="title">
            <string>Agenda</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="Ru_CommonInput">
           <attribute name="title">
            <string>Ru_CommonInput</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_2"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="Te_CommonInput">
           <attribute name="title">
            <string>Te_CommonInput</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_3"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="RumaCreateCustom">
           <attribute name="title">
            <string>RumaCreateCustom</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_4"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>RPV_CommonInput</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_5"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>RCA_CommonInput</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_6"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_8">
           <attribute name="title">
            <string>ParHr_CommonInput</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_7"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_9">
           <attribute name="title">
            <string>MicroSlp_CommonInput</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_8"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_10">
           <attribute name="title">
            <string>CAL_TA_CommonInput</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_9"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>ManualTcConfig</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_10"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>RCA_RS</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_11"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_11">
           <attribute name="title">
            <string>Internal_RS</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_12"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_12">
           <attribute name="title">
            <string>SPD</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_16">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_13"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_13">
           <attribute name="title">
            <string>FCC_RS</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_14"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
